[AGM-1813] Create 'Solving Problems' section on AdGuard for Safari (#416)
AdguardTeam/KnowledgeBase: docs/adguard-for-safari/solving-problems/logs.md

---
title: How to collect logs
sidebar_position: 4
---

:::info

This article is about AdGuard for Safari, which safeguards only your Safari browser. To protect your entire device, [download the AdGuard app](https://agrd.io/download-kb-adblock)

:::

> ## How to collect logs in AdGuard for Safari

Reviewer: и вот тут зачем-то повтор заголовка

Author: тут тоже самое:
@sonya просто хотелось чтобы слева в кб-меню было покороче, а на самой страничке более развернутый(длинный) заголовок, думаешь не надо?

Reviewer: мне кажется не надо, странное дублирование получается

> To analyze and diagnose different problems that may potentially arise, the AdGuard support service might need AdGuard log files. These files contain records about errors and other events that occurr with the software. Below is the brief instruction describing the way to get the log files and send them to the support service if necessary.

Reviewer: occurr хехе

instuctions во множественном всегда

```suggestion
To analyze and diagnose various problems that may occur with AdGuard for Safari, the AdGuard support service may need its log files. These files contain records of errors and other software events. Below are brief instructions on how to collect logs and send them to support if necessary.
```

>  1. Open AdGuard for Safari settings by clicking its icon beside the Safari's search bar.

Reviewer: ```suggestion
 7. Click the AdGuard for Safari icon next to the search bar and open its settings.
```

Reviewer: а еще можно из трея открыть...

Author: ты имеешь в виду Dock или что?

 2. Scroll down and click *Export logs*.

>  3. Choose where do you want to sae the .zip file and click *Save*.

Reviewer: where do you want → where you want
или так:

```suggestion
3. Choose a location to save the .zip file and click *Save*.
```

> Now, you can share that file with our support team (support@adguard.com) or you can report a bug on GitHub:

Reviewer: ```suggestion
Now you can send this file to support@adguard.com or attach it to your bug report on GitHub:
```

Reviewer 2: В логах сафарийного Адгарда не сохраняется ничего _такого_, о чём юзер может пожалеть?

Reviewer: не в курсе, но кажется не должно, там же даже лицуха не нужна

Reviewer 2: логирование доменов может. Advanced Protection вроде способен получать такую информацию.

Author: ну любые логи в наших продуктах что-то такое пишут, если посылает логи то в любом случае доверяет нам какую-то инфу ж

 1. Create an issue on GitHub

>  2. Send the log file to devteam@adguard.com. Include the time of the bug and attach a link to your issue or its number (it appears as #number next to the title).

Reviewer: а вариант прикрепить? 🤔

Reviewer: еще хочется какую-то минимальную инструкцию типа «опишите проблему и прикрепите логи. альтернативные способы, если вы не хотите шерить приватную инфу — отправьте репорт просто так, а потом пришлите ссылку на репорт и логи (или ссылку на гугл диск) на девтим»

Author: хз я взял это в основном из кб по винде. почему интересно там так 🤔
